signac/contrib/project.py

```diff
@@ -52,7 +52,7 @@
 # doc_filter parameters will only be preserved for backwards compatibility but
 # not advertised as part of the API in signac 2.0.
 DOC_FILTER_WARNING = (
-    "The doc_filter argument was deprecated in version 1.7 and will be removed "
+    "The doc_filter argument is deprecated in version 1.7 and will be removed "
```

Comment:
```suggestion
    "The doc_filter argument is deprecated since version 1.7 and will be removed "
```

Reply:
I settled on "The doc_filter argument is deprecated as of version 1.7" because it matches another deprecation in the code base. This will be added in a separate PR that I'm working on, so I'm resolving this conversation.